File: server/SwitchCase/Models/Skin.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SwitchCase.Models
{
    public class Skin
    {
        public int SkinId { get; set; }

        public int WeaponId { get; set; }

        public string Name { get; set; }

        public int CaseId { get; set; }

        public string ImageLink { get; set; }

        public float Price { get; set; }

        public enum RarityEnum { MilSpec, Restricted, Classified, Covert }

        public RarityEnum Rarity { get; set; }

        public double DropRate { get; set; }
    }
}

File: android-client/app/src/main/java/com/artemglotov/switchcase/core/models/Skin.kt
package com.artemglotov.switchcase.core.models

import android.os.Parcelable
import com.artemglotov.switchcase.core.enums.Rarity
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Skin(
    val skinId: Int,
    val name: String,
    val price: Float,
    val weaponName: String,
    val dropRate: Double,
    val caseId: Int,
    val imageLink: String,
    val rarity: Rarity
) : Parcelable